### PR TITLE
add js for active buttons

### PR DESCRIPTION
**Pull request by matthewcorreia-hitfox**

### CODE REVIEW

File: app/assets/javascripts/ui_components/active_buttons.coffee [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$ ->
+  $('.btn-group .btn').click ->
+    if $(this).siblings().hasClass 'active'
```
Comment (RKushnir) [inline app/assets/javascripts/ui_components/active_buttons.coffee]: Is this check necessary? I think, it wouldn't fail if you just execute it anyway

Comment (matthewcorreia-hitfox) [inline app/assets/javascripts/ui_components/active_buttons.coffee]: @RKushnir it's true - i guess i was thinking about performance, having it fail earlier and listen to fewer buttons. but it probably doesn't make much of a difference. i could remove the condition?

Comment (RKushnir) [inline app/assets/javascripts/ui_components/active_buttons.coffee]: Up to you, just a remark

Comment (matthewcorreia-hitfox) [inline app/assets/javascripts/ui_components/active_buttons.coffee]: @RKushnir ok, i'll think a bit more about it. thanks for pointing it out :)

Comment (matthewcorreia-hitfox) [inline app/assets/javascripts/ui_components/active_buttons.coffee]: it was re-worked to not need the js at all anyway :+1:

Comment (RKushnir) [inline app/assets/javascripts/ui_components/active_buttons.coffee]: cool, css FTW

**Pull request merged**